Tryouts/Prototypes/Shell/Node-launcher/Lib/src/WindowConfig.ts

```diff
@@ -1,6 +1,7 @@
 export interface WindowConfig {
     title?: string;
-    url?: string;
+    icon?: string;
+    url: string;
```

Comment:
You can create a `BrowserWindow` without `url` and then call `loadUrl`.

Reply:
True, fixed